<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>391</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Question Selection</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="qestSelection">
      <item>
       <property name="text">
        <string>Q1: ***************</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Q2: ###########</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="questConfirm">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="track_label">
      <property name="text">
       <string>Track</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="PlotWidget" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>611</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
